tests/test_pipeline/test_policy/test_bert_model.py

```diff
@@ -78,9 +78,11 @@ def check_bert_model_policy():
     stage_manager = PipelineStageManager(pg_mesh, PP_DIM)
     rank = dist.get_rank()
 
-    model_policy = BertModelPolicy(stage_manager, len(model.encoder.layer))
+    model_policy = BertModelPolicy(stage_manager)
+    model_policy.set_model(model)
+    model_policy.set_layers_per_stage()
```

Comment:
It seems that  policy class does not have this method

Reply:
This test is useless now, I will fix it.